tests/registry/test_predicate_mapping.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+
+from eth_abi.registry import (
+    PredicateMapping,
+)
+
+
+@pytest.fixture
+def mapping():
+    return PredicateMapping()
+
+
+def gt(x, y):
+    return x > y
+
+
+def lte(x, y):
+    return x <= y
+
+
+def test_can_add_predicates_and_find_corresponding_values(mapping):
+    mapping.add(gt, 'greater than')
+    mapping.add(lte, 'less than or equal')
+
+    assert mapping.find(2, 1) == 'greater than'
+    assert mapping.find(2, 2) == 'less than or equal'
+    assert mapping.find(2, 3) == 'less than or equal'
```

Comment:
This isn't a big deal at all since these are so simple, but traditionally these would be done with the `pytest.mark.parametrize` decorator.  Just making sure you're aware of it's existence/usage.

Reply:
I did notice you guys using that decorator and thought I might use it in a case where there are many repetitions of some assertion.  I can use it there it you'd like.

Reply:
Actually I'm remembering now that the reason I did that was I thought it would be nice to test if the `find` method works if more than one predicate is present in the mapping.